examples/basic/apps/docs/next.config.js

```diff
@@ -1,4 +1,4 @@
+/** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: true,
-  transpilePackages: ["ui"],
```

Comment:
Does next14 no longer need this?

Reply:
We're no longer using next transpile for the basic example, it has it's own build step instead.